xbmc/platform/posix/storage/discs/DiscDriveHandlerPosix.cpp

```diff
@@ -0,0 +1,137 @@
+/*
+ *  Copyright (C) 2022 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#include "DiscDriveHandlerPosix.h"
+
+#include "storage/cdioSupport.h"
+#include "utils/log.h"
+
+#include <memory>
+
+namespace
+{
+constexpr int MaxOpenRetries = 3;
```

Comment:
```suggestion
constexpr int MAX_OPEN_RETRIES = 3;
```
https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#82-constants